packages/fork-choice/test/unit/protoArray/protoArray.test.ts

```diff
@@ -107,10 +107,7 @@ describe("ProtoArray", () => {
     ];
 
     for (const [ancestorRoot, descendantRoot, isDescendant] of assertions) {
-      expect(fc.isDescendant(ancestorRoot, descendantRoot)).to.equal(
-        isDescendant,
-        `${descendantRoot} must be ${isDescendant ? "descendant" : "not descendant"} of ${ancestorRoot}`
-      );
+      expect(fc.isDescendant(ancestorRoot, descendantRoot)).toBe(isDescendant);
```

Comment:
If this tests fails the err message will be hard to reason about compared to previous (custom) assertion err message